### PR TITLE
initial ory route

### PR DESCRIPTION
**Pull request by d-g-town**

### CODE REVIEW

File: dashboard/src/main/MainWrapper.tsx [hunk 1]
```diff
@@ -7,22 +8,30 @@ import MainWrapperErrorBoundary from "shared/error_handling/MainWrapperErrorBoun
...
-        <AuthnProvider>
-          <MainWrapperErrorBoundary>
-            <Main />
-          </MainWrapperErrorBoundary>
-        </AuthnProvider>
-      </AuthzProvider>
+      <ThemeProvider themeOverrides={{}}>
+        {/* We dont need to pass any custom translations */}
+        {/* <IntlProvider> */}
+        {/* We pass custom translations */}
```
Comment (ferozemohideen) [inline dashboard/src/main/MainWrapper.tsx:22]: what does this mean

File: dashboard/src/main/auth/OryLogin.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  // Get the flow based on the flowId in the URL (.e.g redirect to this page after flow initialized)
+  const getFlow = useCallback(
+    async (flowId: string) =>
+      await sdk
+        // the flow data contains the form fields, error messages and csrf token
+        .getLoginFlow({ id: flowId })
+        .then(({ data: flow }) => {
+          setFlow(flow);
+        })
+        .catch(sdkErrorHandler),
```
Comment (ferozemohideen) [inline dashboard/src/main/auth/OryLogin.tsx:65]: consider switching these to async/await syntax for consistency

**Review by ferozemohideen — APPROVED** [review]
lots of error states here - for testing, if you haven't already, i would recommend actually throwing new Error() objects in the code wherever appropriate and making sure that the error states / messaging makes sense

File: dashboard/src/main/MainWrapper.tsx (hunk 1 above)
Comment (d-g-town) [reply dashboard/src/main/MainWrapper.tsx:22]: removed

**Review by ianedwards — APPROVED** [review]

File: dashboard/src/shared/auth/sdk.ts [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+  new Configuration({
+    basePath,
+    // we always want to include the cookies in each request
+    // cookies are used for sessions and CSRF protection
+    baseOptions: {
+      withCredentials: true,
+    },
+  })
+);
+
```
[the remark this answers is not in the document]
Comment (d-g-town) [reply dashboard/src/shared/auth/sdk.ts]: removed, thanks

**Pull request merged**